chef/cookbooks/provisioner/templates/default/autoyast.xml.erb

```diff
@@ -268,7 +268,7 @@ sync
       <package>biosdevname</package>
       <package>netcat</package>
       <package>ruby2.1-rubygem-chef</package>
-<% if @is_ses -%>
+<% if @is_ses and @target_platform_version == "12.0" -%>
```

Comment:
That was the code I was thinking of: https://github.com/crowbar/barclamp-provisioner/blob/master/chef/cookbooks/provisioner/recipes/update_nodes.rb#L253

And indeed, there's no check for the version there.

@tserong I guess the "problem" is that it will more or less always work, but in theory, the test is not perfect ;-)